sdk/face/azure-ai-vision-face/CHANGELOG.md

```diff
@@ -4,12 +4,25 @@
 
 ### Features Added
 
+- Added support for the Large Face List and Large Person Group:
+  - Added operation groups `LargeFaceListOperations` and `LargePersonGroupOperations` to `FaceAdministrationClient`.
+  - Added operations `find_similar_from_large_face_list`, `identify_from_large_person_group` and `verify_from_large_person_group` to `FaceClient`.
```

Comment:
nit: please remember update `README.md`

Reply:
We released new codegen version and please update to latest main branch then regenerate SDK. After that, the pylint issues reported by CI shall be fixed. @Han-msft